cmd/tls_test.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/arschles/assert"
+	"github.com/deis/controller-sdk-go/api"
+	"github.com/deis/workflow-cli/pkg/testutil"
+)
+
+func TestTLSInfo(t *testing.T) {
+	t.Parallel()
+	cf, server, err := testutil.NewTestServerAndClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+	var b bytes.Buffer
+	cmdr := DeisCmd{WOut: &b, ConfigFile: cf}
+
+	server.Mux.HandleFunc("/v2/apps/numenor/tls/", func(w http.ResponseWriter, r *http.Request) {
+		testutil.SetHeaders(w)
+		fmt.Fprintf(w, `{
+	"uuid": "c4aed81c-d1ca-4ff1-ab89-d2151264e1a3",
+	"app": "numenor",
+	"owner": "nazgul",
+	"created": "2016-08-22T17:40:16Z",
+	"updated": "2016-08-22T17:40:16Z",
+	"https_enforced": true
+}`)
+	})
+
+	err = cmdr.TLSInfo("numenor")
+	assert.NoErr(t, err)
+	assert.Equal(t, b.String(), `=== numenor TLS
+HTTPS Enforced: true
+`, "output")
+}
+
+func TestTLSEnable(t *testing.T) {
+	t.Parallel()
+	cf, server, err := testutil.NewTestServerAndClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+	var b bytes.Buffer
+	cmdr := DeisCmd{WOut: &b, ConfigFile: cf}
+
+	server.Mux.HandleFunc("/v2/apps/numenor/tls/", func(w http.ResponseWriter, r *http.Request) {
+		testutil.SetHeaders(w)
+		body, err := ioutil.ReadAll(r.Body)
```

Comment:
I'm asserting that the tests included data in the body so that we can fail sooner and identify the issue.

Once this is rebased after https://github.com/deis/controller-sdk-go/pull/92 is merged, I will use the helper function. Thanks